### ISSUE TITLE
Make error output less verbose

### ISSUE THREAD

**Issue by Chi-teck** [drush-ops/drush#4039]
When the error is caused by wrong input Drush should only print relevant message unless more verbose output was requested via `-v` option.

Example:
```
$ drush sql-query 'WRONG QUERY'

In SqlCommands.php line 194:
                 
  Query failed.  
                 

sql:query [--result-file [RESULT-FILE]] [--file FILE] [--file-delete] [--extra EXTRA] [--db-prefix] [-h|--help] [-q|--quiet] [-v|vv|vvv|--verbose] [-V|--version] [--ansi] [--no-ansi] [-n|--no-interaction] [-d|--debug] [-y|--yes] [--no] [--remote-host REMOTE-HOST] [--remote-user REMOTE-USER] [-r|--root ROOT] [-l|--uri URI] [--simulate] [--pipe] [-D|--define DEFINE] [--database [DATABASE]] [--target [TARGET]] [--db-url DB-URL] [--notify [NOTIFY]] [--xh-link XH-LINK] [--druplicon] [--] <command> [<query>]
```

In this case `In SqlCommands.php line 194:`  and `sql:query` signature are not related to the error. The only relevant information that should be printed is `Query failed`. The current output is confusing.

A user may provide `-v` option to increase the verbosity of messages. However in this case it does not work well. Drush prints stack trace instead of PDO exception message which would help the user to fix the query.

Worth noting, this works pretty well in Drush 8.

**Comment by Chi-teck** [comment]
Note that the `sql:query` is used just for example. The problem is relevant to many other commands.

**Comment by weitzman** [comment]
I think I agree. That long ceremony listing all args+options for the current command comes from Symfony Console. I suppose someone could look into suppressing it.

**Comment by weitzman** [comment]
Fixed in #4652
